flink-table/flink-sql-gateway/src/test/java/org/apache/flink/table/gateway/service/MaterializedTableStatementITCase.java

```diff
@@ -652,6 +654,158 @@ void testAlterMaterializedTableWithoutSavepointDirConfigured() throws Exception
+        Trigger.TriggerState suspendTriggerState =
+                embeddedWorkflowScheduler
+                        .getQuartzScheduler()
+                        .getTriggerState(new TriggerKey(workflowName, workflowGroup));
+
+        assertThat(suspendTriggerState).isEqualTo(Trigger.TriggerState.PAUSED);
+
+        // resume materialized table
+        String alterMaterializedTableResumeDDL =
+                "ALTER MATERIALIZED TABLE users_shops RESUME WITH ('debezium-json.ignore-parse-errors' = 'true')";
+        OperationHandle alterMaterializedTableResumeHandle =
+                service.executeStatement(
+                        sessionHandle, alterMaterializedTableResumeDDL, -1, new Configuration());
+        awaitOperationTermination(service, sessionHandle, alterMaterializedTableResumeHandle);
+
+        ResolvedCatalogMaterializedTable resumedCatalogMaterializedTable =
+                (ResolvedCatalogMaterializedTable)
+                        service.getTable(
+                                sessionHandle,
+                                ObjectIdentifier.of(
+                                        fileSystemCatalogName,
+                                        TEST_DEFAULT_DATABASE,
+                                        "users_shops"));
+
+        assertThat(resumedCatalogMaterializedTable.getOptions())
+                .doesNotContainKey("debezium-json.ignore-parse-errors");
+        assertThat(resumedCatalogMaterializedTable.getRefreshStatus())
+                .isEqualTo(CatalogMaterializedTable.RefreshStatus.ACTIVATED);
+
+        // verify workflow is resumed
+        refreshHandler = resumedCatalogMaterializedTable.getSerializedRefreshHandler();
+        EmbeddedRefreshHandler resumeRefreshHandler =
+                EmbeddedRefreshHandlerSerializer.INSTANCE.deserialize(
+                        refreshHandler, getClass().getClassLoader());
+
+        assertThat(resumeRefreshHandler.getWorkflowName()).isEqualTo(workflowName);
+        assertThat(resumeRefreshHandler.getWorkflowGroup()).isEqualTo(workflowGroup);
+
+        JobDetail jobDetail = embeddedWorkflowScheduler.getQuartzScheduler().getJobDetail(jobKey);
+        Trigger.TriggerState resumedTriggerState =
+                embeddedWorkflowScheduler
+                        .getQuartzScheduler()
+                        .getTriggerState(new TriggerKey(workflowName, workflowGroup));
+        assertThat(resumedTriggerState).isEqualTo(Trigger.TriggerState.NORMAL);
+
+        WorkflowInfo workflowInfo =
+                fromJson((String) jobDetail.getJobDataMap().get(WORKFLOW_INFO), WorkflowInfo.class);
+        assertThat(workflowInfo.getDynamicOptions())
+                .containsEntry("debezium-json.ignore-parse-errors", "true");
+
+        // delete the workflow
+        embeddedWorkflowScheduler.deleteScheduleWorkflow(jobKey.getName(), jobKey.getGroup());
+    }
+
+    private void createMaterializedTableInFullMode(String tableName, List<Row> data)
```

Comment:
Why don't you reuse the util method `createAndVerifyCreateMaterializedTableWithData`?

Reply:
createAndVerifyCreateMaterializedTableWithData creates a table in continous mode, I made some modifications to support full mode.